docs/frontend.md

```diff
@@ -175,6 +175,14 @@ and independent view components, rendered server-side.
 
 For more information, refer to the [components `README.md`](../app/components/README.md).
 
+To preview components and their available options, we use [Lookbook](https://lookbook.build/) to
+generate a navigable index of our available components. These previews are available at the [`/components/` route](http://localhost:3000/components/)
+in local development, review applications, and in the `dev` environment. When adding a new component
```

Comment:
Is it worth noting that the `component_previews_enabled` feature flag toggles visibility on/off?

Reply:
I think we could, but I can't personally imagine much need to change it from its defaults, so didn't want to dilute the critical information here.

Do you think there's a use-case to be turning it on and off?

Reply:
Eh, probably not.